Source/WebCore/fileapi/FileReaderLoader.cpp

```diff
@@ -89,6 +89,11 @@ void FileReaderLoader::start(ScriptExecutionContext* scriptExecutionContext, con
         failed(ExceptionCode::SecurityError);
         return;
     }
+
+    auto* contentSecurityPolicy = scriptExecutionContext->contentSecurityPolicy();
```

Comment:
`CheckedPtr contentSecurityPolicy = scriptExecutionContext->contentSecurityPolicy();`

Reply:
hmm.. it doesn't build for `CheckedPtr` type. I can change it to `ContentSecurityPolicy*` type to be consistent.

Reply:
You may need to add `#include "ContentSecurityPolicy.h"`. It's not about specifying the type, but using smart pointers if possible.